### PR TITLE
Refactor storage-system pool capacity data source and documentation

### PR DESCRIPTION
**Pull request by Diptipowervs**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/d/pi_storage_pool_capacity.markdown [hunk 1]
```diff
@@ -47,4 +43,4 @@ In addition to all argument reference list, you can access the following attribu
 - `max_allocation_size` - (Integer) Maximum allocation storage size (GB).
 - `storage_type` - (String) Storage type of the storage pool.
 - `total_capacity` - (Integer) Total pool capacity (GB).
-- `replication_enabled` - (Boolean) Replication status of the storage pool.
```
Comment (ismirlia) [inline website/docs/d/pi_storage_pool_capacity.markdown:50]: `replication_enabled` is sorted incorrect. I must have missed sorting it alphabetically.

It should go between `max_allocation_size` and `storage_type`

File: website/docs/d/pi_storage_pools_capacity.markdown [hunk 2]
```diff
@@ -56,4 +52,4 @@ In addition to all argument reference list, you can access the following attribu
   - `pool_name` - (String) The pool name.
   - `storage_type` - (String) Storage type of the storage pool.
   - `total_capacity` - (Integer) Total pool capacity (GB).
-  - `replication_enabled` - (Boolean) Replication status of the storage pool.
```
Comment (ismirlia) [inline website/docs/d/pi_storage_pools_capacity.markdown:59]: Same issue here as above.

File: website/docs/d/pi_storage_pool_capacity.markdown (hunk 1 above)
Comment (Diptipowervs) [reply website/docs/d/pi_storage_pool_capacity.markdown:50]: Addressed

**Comment by michaelkad** [comment]
@Diptipowervs could you resolve `ibm_pi_constants.go` conflicts?

**Comment by Diptipowervs** [comment]
Pr raised in main repo

**Pull request closed without merging**